src/docs/software/using/R.md

```diff
@@ -202,6 +202,57 @@ Loading required package: parallel
 >
 ```
 
+#### Package dependencies
+
+Sometimes when installing R packages, other software is needed for the installation and/or compilation.  For example the package `sf`.  If you try to install it:
+
+```
```

Comment:
Do you want to add the language to the blocks so it uses syntax highlighting?

Reply:
sorry, you mean it should be double back ticks? ``sf``  feel free to make any changes if you like

Reply:
I can give a quick example - for most of these templates they have syntax highlighting built in, and you use it by way of adding the language after the ticks. Here is without it:

```
echo "Hello World"
```
and with it
```bash
echo "Hello World"
```
or for R, with and without:

```
stuff = c("1", "2", "3")
install.packages("pancakes")
```

```R
stuff = c("1", "2", "3")
install.packages("pancakes")
```
Click on edit to view the source and see how they differ! It's a subtle detail, but I think the added color accent tends to make the docs look a lot better than without.